_posts/2017-01-09-Jak-psat-php-Zavislosti+verzovaci-systemy+debugging.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Jak psát PHP? (2/3) - Závislosti + verzovací systémy + debugging
+---
+
+# Manažer pro závislosti - Composer
+[Composer](https://getcomposer.org/) je software řešící závislosti v kódu. V podstatě se stará o to, aby všechny balíčky cizího kódu (na kterých závisíte, proto závislosti) byly stažené, ve správné verzi a nebily se mezi sebou. Composer je umí sám stahovat, komtrolovat nové verze i bezpečně odstraňovat. V PHP dlouho něco takového chybělo a všichni jsme rádi, že tu je (kdo z vás vzpomíná na stahování knihoven z php.net při rozcházení cizího projektu?). 
```

Comment:
"kdo z vás vzpomíná na stahování knihoven z php.net při rozcházení cizího projektu?"

To bych možná vyhodil, když je to pro začátečníky. Tady by se mohli cítit mimo, příliš mladí.